asar-teensy-firmware/TestScripts/Combined/MotorandProximity/MOTOR.cpp

```diff
@@ -0,0 +1,108 @@
+/* Motor.cpp
+ *  Author: Aaron Mallabar
+ *  Date:   January 22,2018
+ *  Description: Low Level Motor Control
+ *  
+ */
+ 
+
+//Teensy Pinout
+//RightA is forward
+//LeftB is forward
+
+#include "Arduino.h"
+#include "MOTOR.h"
+
+
+namespace ASAR
+{
```

Comment:
Add a line of whitespace for clarity after the open brace.

Reply:
Done